IOSAccessAssessment/Views/SetupView.swift

```diff
@@ -8,11 +8,10 @@
 import SwiftUI
 
 struct SetupView: View {
-    let classes = ["Background", "Aeroplane", "Bicycle", "Bird", "Boat", "Bottle", "Bus", "Car", "Cat", "Chair", "Cow", "Diningtable", "Dog", "Horse", "Motorbike", "Person", "Pottedplant", "Sheep", "Sofa", "Train", "TV"]
     @State private var selection = Set<Int>()
     
     var body: some View {
-        NavigationView {
+        NavigationStack {
```

Comment:
This may not be the best practice because NavigationStack was introduced very recently.

Reply:
iOS 16+